src/view/TextAreaView.java

```diff
@@ -46,12 +56,46 @@ public TextAreaView(TextAreaViewModel textAreaViewModel) {
 
         // Add the mainPanel to the CENTER of TextAreaView
         this.add(mainPanel, BorderLayout.CENTER);
+
+        clearButton.addActionListener(new ActionListener() {
+            final ClearViewModel clearViewModel = new ClearViewModel();
+
+            final ClearOutputBoundary clearOutputBoundary = new ClearPresenter(clearViewModel, textArea);
```

Comment:
ClearPresenter and ClearInteractor cannot be directly created from TextAreaView as it violates CA